BusinessLayer/LoggerService/LoggerService.cpp.bak

```diff
@@ -0,0 +1,174 @@
+#include <QDebug>
```

Comment:
Please remove this file. `.bak` files come from running AStyle and are the original files before being edited. Usually these are removed in the `format.sh`, so I'm not really sure how this is here O_o